### PR TITLE
Spec updates from Opcode and Consistency Review

### PR DESCRIPTION
**Pull request by aswaterman**
- Update instruction-listing tables to match mailing-list discussions
- Rename *u.w to *.uw, on the basis that these instructions behave materially different from the existing *.w instructions.
- Rename sb* to b*, on the basis that the "sb" prefix looks too much like the "store byte" instruction, and "b" by itself already implies "single bit".
- Rename pcnt* to cpop*, to use the same "c" prefix as ctz/clz.
- Bump document version to v0.93.

Supersedes #99

### CODE REVIEW

**Comment by kito-cheng** [comment]
bext seems like has two means, single bit extract in zbs and (multiple) bit extract in zbe, I guess it's kind of mistake?

**Comment by aswaterman** [comment]
You and/or Nelson should respond to Ken's email on the tech-bitmanip list with this observation.

It's not super urgent since we aren't freezing Zbs or Zbe yet.

**Comment by kito-cheng** [comment]
> You and/or Nelson should respond to Ken's email on the tech-bitmanip list with this observation.
> 
> It's not super urgent since we aren't freezing Zbs or Zbe yet.

Got it, I reply the thread on tech-bitmanip.

**Comment by aswaterman** [comment]
Thank you!

File: texsrc/bext.tex [hunk 1]
```diff
@@ -1985,16 +1985,16 @@ \section{Opcode Encodings}
...
+|  0010000    |   rs2   |   rs1   | 100 |    rd   |   0111011   |  + SH2ADD.UW
+|  0010000    |   rs2   |   rs1   | 110 |    rd   |   0111011   |  + SH3ADD.UW
 |---------------------------------------------------------------|
 |  0000101    |   rs2   |   rs1   | 101 |    rd   |   0110011   |  - MAX
 |  0000101    |   rs2   |   rs1   | 110 |    rd   |   0110011   |  - MINU
 |  0000101    |   rs2   |   rs1   | 110 |    rd   |   0110011   |  + MAX
 |  0000101    |   rs2   |   rs1   | 101 |    rd   |   0110011   |  + MINU
 |---------------------------------------------------------------|
-|  0100100    |   rs2   |   rs1   | 000 |    rd   |   0111011   |  - SUBU.W
+|  0100100    |   rs2   |   rs1   | 000 |    rd   |   0111011   |  - SUB.UW
```
Comment (kito-cheng) [inline texsrc/bext.tex:1997]: subu.w already gone before, and here is just a history record, so I guess this should not change?

Comment (aswaterman) [reply texsrc/bext.tex:1997]: Thanks for catching this.  As you can see, I was just using `sed`...

I think it should just be removed.  It isn't necessary to leave it here for history; that is what the commit history is for :)

Comment (aswaterman) [reply texsrc/bext.tex:1997]: Oops, I now see that it is actually an explicit history log.  My mistake.  I'll just revert the change.

**Comment by aswaterman** [comment]
The name conflict is resolved as part of PR #102 (which will be versioned v0.94-draft).

**Comment by aswaterman** [comment]
Ken Dockser authorized merging this PR in his capacity as chair.  (He's still getting his github write permissions set up.)

**Comment by kito-cheng** [comment]
@aswaterman Do you mind push a tag `v0.93` to the repo, I saw other versions has corresponding tag there.

**Comment by aswaterman** [comment]
Done.  The Release also has the PDF.

**Comment by kito-cheng** [comment]
Thanks, and here is another request, could you mention zba, zbb and zbc with version `0.93` is frozen in this repo or https://github.com/riscv/riscv-isa-manual?

**Comment by aswaterman** [comment]
That's already done:
https://github.com/riscv/riscv-bitmanip/blob/29947eb5adb5bc18b23b2f8c7cac6816df8c1af0/texsrc/bext.tex#L79

(Those words mean the same thing as "frozen".)

**Comment by kito-cheng** [comment]
Hmmmm, I expect there is should be a explicit `frozen` word here, since that's kind of official term in RISC-V?

**Comment by aswaterman** [comment]
I don’t think it makes a difference. Feel free to make a PR, but we already
released the PDF, so the PDF will continue to use these words.

**Comment by jnk0le** [comment]
@aswaterman so that means Zba was frozen without resolving #90?

**Comment by aswaterman** [comment]
Apparently, but a non-resolution to #90 is an appropriate response.  Providing shifts up to 3 is about circuit delay; 3 is similar to 2, but 4 adds another level of logic.  And supporting shifts by 3 is still plenty useful in RV32 implementations.  While I would argue against providing shadd4, it can always be provided as part of a future extension; that decision does not need to be made now.

**Pull request merged**